src/azul/openapi/__init__.py

```diff
@@ -150,24 +57,12 @@ def get_app_specs(app):
     """
     Extract OpenAPI specs from a Chalice app object.
     """
-    path_specs = {}
-    method_specs = {}
-    for entries in app.routes.values():
-        func = next(iter(entries.values())).view_function
-        func_path_specs = getattr(func, 'path_specs', None)
-        if func_path_specs:
-            for path, spec in func_path_specs.items():
-                if path in path_specs:
-                    assert spec == path_specs[path], f'Path spec for {spec} already exists'
-            path_specs.update(func.path_specs)
-        else:
-            # TODO (jesse): raise if route is undocumented
-            pass
-        func_method_specs = getattr(func, 'method_specs', None)
-        if func_method_specs:
-            method_specs.update(func_method_specs)
+    undocumented = app.routes.keys() - app.path_specs.keys()
+    if undocumented:
+        # TODO (jesse): raise if route is undocumented
```

Comment:
@jessebrennan is there a ticket for this? If so, post that here so @noah-aviel-dove can add a reference.

Reply:
… and if not create one and post ticket number here.

Reply:
In my head this would be taken care of in #1450.

Reply:
The ticket reference still needs to be added here.